EAD/EAD.tex

```diff
@@ -50,27 +56,27 @@
 
 
 \begin{document}
-
+\maketitle
 
 \section{Introduction}
 This group will create an integration of the \gls{Gamygdala} engine in the programming language \gls{GOAL}.
 \subsection{Goals}
-In this project we have multiple goals. Besides the goals between the different groups that can be found in the product vision document we also have goals that are group specific. We think that it is very important for us that we have good code quality. Also functionality is important.By functionality we mean that focussing on
+In this project we have multiple goals. Besides the goals between the different groups that can be found in the product vision document we also have goals that are group specific. We think that it is very important for us that we have good code quality. Also functionality is important with this we mean focusing on creating code that works well and that has the necessary features to bring this proof of concept to a good end.
 
 \subsubsection{Source code quality}
-We think that it is important to have readible code that is well written with the correct design patterns. Other software engineering principles we have learned so far should also be used appropriately and correctly. For example: Method sizes.
+We think that it is important to have readable code that is well written with the correct design patterns. Other software engineering principles we have learned so far should also be used appropriately and correctly. For example: Method sizes.
 
 \subsubsection{Functionality}
-By testing in different ways we want our code to be very functional. Since our job is to create an integration, functionality is the most important goal for our team.
+By testing in different ways we want our code to have all the necessary features working well. Since our job is to create an integration, functionality is the most important goal for our team. It should be noted though that this is a proof of concept, so while we will focus on making sure the other groups receive the necessary features some of them might be implemented in a less nice way in order to be able to implement them quickly. An example of this is the emotion configuration file. Instead of adding the definitions of this file to the agent and MAS files we decided to use a separate file since this meant we could more easily create our own parser for it since adapting the files of GOAL would require learning a lot about \gls{ANTLR}.
```

Comment:
Ik zou misschien eerder means ipv meant gebruiken. Opzich niet fout, alleen best alijd in zelfde tijd blijven schrijven. Zelf doe ik het ook niet altijd maar ze zeggen dat het eigenlijk het beste zijn.